.github/workflows/build-push-image.yml

```diff
@@ -0,0 +1,31 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+      - 'pearson-release/olive.stage'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Build and push Image
+        uses: Pearson-Advance/tutor-build-image-action@0.1.1
+        with:
+          python-version: ${{ vars.PYTHON_VERSION }}
+          tutor-version: ${{ vars.TUTOR_VERSION }}
+          path-image-to-be-built: '/env/build/openedx'
+          repository-name: ${{ vars.REPOSITORY_NAME }}
+          image-name: ${{ vars.IMAGE_NAME }}
+          image-tag: ${{ vars.IMAGE_TAG }}
+          docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
+          docker-token: ${{ secrets.DOCKERHUB_TOKEN }}
+          no-docker-cache: true
+          gh-access-token: ${{ secrets.GH_PAT }}
+          theme-repository: "https://github.com/Pearson-Advance/openedx-themes"
+          theme-branch: ${{ vars.THEME_BRANCH }}
+          extra-private-requirements: true
+          private-repositories: ${{ vars.PRIVATE_REPOSITORIES }}
+          branches: ${{ vars.branches }}
```

Comment:
Missing endline

Reply:
@alexjmpb thanks!

Reply:
@sergivalero20 With this i meant to do it for all variables, with the objective of like categorize these variables for the building process

Reply:
@alexjmpb you mean that adjust the variables names adding a prefix as BUILD?

Reply:
@sergivalero20 yes what do you think?

Reply:
Ok, I agree.
Suggestion applied.